Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "FINUFFT"
 uuid = "d8beea63-0952-562e-9c6a-8e8ef7364055"
 author = "Ludvig af Klinteberg <ludvigak@kth.se>"
-version = "2.1.0"
+version = "3.0.0"
```

Comment:
Are you bumping up the major number since it changes the interface (a bit)?  It still is an interface to FINUFFT 2.0.3.  I wonder if that is confusing? I guess no more than right now :)

Reply:
Yep, just released v3.0. We'll just have to live with the slight weirdness of package and interface running at two different versions